test/src/tests/glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: tests
 import:
 - package: github.com/Azure/go-autorest
-  version: ^9.5.1
+  version: ^10.1.0
```

Comment:
> version: ^10.1.0 [](start = 1, length = 17)

when we pull mater in dev lets not forget to update dep to 10 also.

Reply:
Yep there will be a merge conflict here due to migrating to dep in master, I'll update the version when I resolve the conflict.